dispatch/apps/events/event_integrations.py

```diff
@@ -92,3 +100,63 @@ def get_image(self):
             raise FacebookEventError(FACEBOOK_ERROR_MESSAGE)
 
         return image_url
+
+class UBCEvent(object):
+    """UBC Event class"""
```

Comment:
Make this a bit more descriptive -- something like "A class to scrape event information from UBC event webpages."